### PR TITLE
[MDEP-712] update JIRA URL

### PR DESCRIPTION
**Pull request by elharo**
@slachiewicz

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -45,7 +45,7 @@
   </scm>
   <issueManagement>
     <system>jira</system>
-    <url>https://issues.apache.org/jira/browse/MSHARED/component/12326428</url>
+    <url>https://issues.apache.org/jira/browse/MSHARED-932?jql=project%20%3D%20MSHARED%20AND%20component%20%3D%20maven-dependency-analyzer</url>
```
Comment (pzygielo) [inline pom.xml:48]: Could it point to JIRA without reference to specific issue, like MSHARED-932 here?

Comment (michael-o) [reply pom.xml:48]: Agree, this needs to be changed.

Comment (elharo) [reply pom.xml:48]: What URL do you suggest? So far I have not been able to construct one that works and does not include an issue.

Comment (pzygielo) [reply pom.xml:48]: > What URL do you suggest? So far I have not been able to construct one that works and does not include an issue.

Similar to one proposed in https://github.com/apache/maven-project-utils/pull/3#discussion_r453213245

Comment (elharo) [reply pom.xml]: done

**Pull request merged**